bench/run-hjo.sh

```
@@ -0,0 +1,33 @@
+PK_Telluride="2c65940725bbf10b452197fba41c6cb14afd41e28e0be22aab49bf246b0c84e3"
+
+SESSIONS=5
+LOGDIR="hjo-results"
+mkdir -p "$LOGDIR"
+
+for session in $(seq 1 $SESSIONS); do
+  for name in $NAMES; do
+    eval "pk=\$PK_${name}"
+    logfile="$LOGDIR/${name}_s${session}.log"
+    echo "=== Session $session: $name ==="
+    deno task bench "$pk" --algorithms "$ALGOS" $COMMON > "$logfile" 2>&1 || true
```

Shellcheck (0.11.0)

[warning] pk is referenced but not assigned.

(SC2154)